src/components/atoms/Countdown/Countdown.js

```diff
@@ -0,0 +1,55 @@
+import React, { useEffect, useState } from 'react'
+
+import './Countdown.scss'
+
+const diff = (a, b) => {
+  const distance = a - b
+
+  return [
+    Math.floor(distance / (1000 * 60 * 60 * 24)),
+    Math.floor((distance % (1000 * 60 * 60 * 24)) / (1000 * 60 * 60)),
+    Math.floor((distance % (1000 * 60 * 60)) / (1000 * 60)),
+    Math.floor((distance % (1000 * 60)) / 1000),
+  ]
```

Comment:
NIT: We could return object instead of array

```
return {
    day: Math.floor(distance / (1000 * 60 * 60 * 24)),
    hour: Math.floor((distance % (1000 * 60 * 60 * 24)) / (1000 * 60 * 60)),
    min: Math.floor((distance % (1000 * 60 * 60)) / (1000 * 60)),
    sec: Math.floor((distance % (1000 * 60)) / 1000),
  }
```
Use it like:

```
const differnce = diff(+new Date(endTime), now)

Object.entries(differnce)
```

Reply:
great refactor! Thanks @PrashantKIsHere